decidim-assemblies/lib/decidim/assemblies/admin_engine.rb

```diff
@@ -30,6 +30,12 @@ class AdminEngine < ::Rails::Engine
 
           resources :attachment_collections, controller: "assembly_attachment_collections"
           resources :attachments, controller: "assembly_attachments"
+
+          resource :export, controller: "assembly_exports", only: :create
+
+          collection do
```

Comment:
Do we need to wrap the `resources :imports, ...`  inside this `colleciton` block?

Reply:
Yes, otherwise the slug would be '/admin/assemblies/:id/import/new' instead of '/admin/assemblies/import/new'.